sdk/js-cloud-server/tsconfig.lib.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "module": "commonjs",
+        "outDir": "../../dist/out-tsc",
+        "declaration": true,
+        "lib": ["esnext", "webworker"],
+        "types": ["node", "@cloudflare/workers-types"]
```

Comment:
if this is meant to be platform agnostic why is it bringing in webworker and cloudflare worker types?

Reply:
That was the theory.

Reply:
?

Reply:
webworker is the other environment type I am specifically trying to support here.

Reply:
yeah but nothing about this library is specific to webworkers or cloudflare so I'm confused why it's relying on those types